lib/login_screen.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'utils/constants.dart';
+class LoginScreen extends StatefulWidget{
+  @override
+  _LoginScreenState createState() => _LoginScreenState();
+}
+
+class _LoginScreenState extends State<LoginScreen> {
+
+  //Text Editing controllers
+  TextEditingController _emailController=TextEditingController();
+  TextEditingController _passwordController=TextEditingController();
+
+  //Colors
+  Color textField_fill_color = const Color(0xFFEBEBEB);
+  Color black_heading=const Color(0xFF000000);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: primary_color,
+      body: SingleChildScrollView(
+        child: SafeArea(
+          minimum: EdgeInsets.all(20.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              SizedBox(height:35.0),
+              RichText(text: TextSpan(
+                  style: TextStyle(fontSize: 58,color: black_heading,fontWeight: FontWeight.w900,letterSpacing: 2.0),
+                  children:[
+                    TextSpan(text:'Hello \nThere '),
+                    TextSpan(text:'.',style:TextStyle(color: secondary_color,fontWeight: FontWeight.w900)),
```

Comment:
Use the Poppins Font family given in the assets folder

Reply:
Is it okay if I set it using ThemeData widget in main.dart? that way it will be applied to the entire app

Reply:
Yup you can go ahead!